source/documentation/how-to/hosted-engine.html.md

```diff
@@ -144,15 +144,40 @@ Assuming you have already deployed Hosted Engine on your hosts and running the H
 
 1.  Set hosted engine maintenance mode to global (now ha agent stop monitoring engine-vm, you can see above how to activate it)
 2.  Access to engine-vm and upgrade oVirt to latest version using the same procedure used for non hosted engine setups.
-3.  Select one of the hosted-engine nodes (hypervisors) and put it into maintenance mode from the engine. Note that the host must be in maintenance to allow upgrade to run.
-4.  Upgrade that host with new packages (changes repository to latest version and run yum update -y) on this stage may appear vdsm-tool exception <https://bugzilla.redhat.com/show_bug.cgi?id=1088805>
-5.  Restart vdsmd (# service vdsmd restart)
-6.  Restart ha-agent and broker services (# systemctl restart ovirt-ha-broker && systemctl restart ovirt-ha-agent)
-7.  Exit the global maintenance mode: in a few minutes the engine VM should migrate to the fresh upgraded host cause it will get an higher score
-8.  When the migration has been completed re-enter into global maintenance mode
-9.  Repeat step 3-6 for all the other hosted-engine hosts
-10. Enter for example via UI to engine and change 'Default' cluster (where all your hosted hosts seats) compatibility version to current version (for example 3.6 and activate your hosts (to get features of the new version)
-11. Change hosted-engine maintenance to none, starting from 3.4 you can do it via UI(right click on engine vm, and 'Disable Global HA Maintenance Mode')
+3.  Host: Enable global maintenance mode from one of the hosts
+```
+host ~ # hosted-engine --set-maintenance --mode=global
```

Comment:
You should do this from the engine, not from the host. See also e.g. https://bugzilla.redhat.com/show_bug.cgi?id=1468875#c1 .

In the distant past, there was no connection at all. Now, if you move host to maintenance from the engine, it also does that for hosted-engine-ha: https://bugzilla.redhat.com/show_bug.cgi?id=1047649

But the other way deliberately does not work.

Reply:
I would have thought this too, but there is no `hosted-engine` command in the engine, that command only exists on hosts (at least in 4.2.1):

```shell
[root@int-engine-01 ~]# rpm -qa | grep ovirt-engine-tools-4
ovirt-engine-tools-4.2.1.6-1.el7.centos.noarch

[root@int-engine-01 ~]# which hosted-engine
/usr/bin/which: no hosted-engine in (/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/root/bin)
```

Reply:
Just to confirm I understand your comment @didib:

- You're agreeing with me and confirming that the MR/patch is actually correct and the global maintenance must be set from the host, not the engine on a hosted-engine deployment? (Which this howto is specific to)